App crashes on pressing home icon on bottomBar

**Actual Behaviour**

App crashes on pressing home button on bottomnavigationbar in starting page.

**Expected Behaviour**

No crash should occur

**Steps to reproduce it**

open app. switch to camera or accounts page then switch to home(gallery) by pressing home button in the middle (not by pressing back button)

**LogCat for the issue**

```
E/AndroidRuntime: FATAL EXCEPTION: main
                                                                      Process: org.fossasia.phimpme, PID: 15263
                                                                      java.lang.RuntimeException: Unable to start activity ComponentInfo{org.fossasia.phimpme/org.fossasia.phimpme.leafpic.activities.LFMainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.os.BaseBundle.getBoolean(java.lang.String)' on a null object reference
                                                                          at android.app.ActivityThread.performLaunchActivity(ActivityThread.java)
                                                                          at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java)
                                                                          at android.app.ActivityThread.-wrap12(ActivityThread.java)
                                                                          at android.app.ActivityThread$H.handleMessage(ActivityThread.java)
                                                                          at android.os.Handler.dispatchMessage(Handler.java)
                                                                          at android.os.Looper.loop(Looper.java)
                                                                          at android.app.ActivityThread.main(ActivityThread.java)
                                                                          at java.lang.reflect.Method.invoke(Native Method)
                                                                          at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java)
                                                                          at com.android.internal.os.ZygoteInit.main(ZygoteInit.java)
                                                                       Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.os.BaseBundle.getBoolean(java.lang.String)' on a null object reference
                                                                          at org.fossasia.phimpme.leafpic.activities.LFMainActivity.onCreate(LFMainActivity.java:189)
                                                                          at android.app.Activity.performCreate(Activity.java)
                                                                          at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java)
                                                                          at android.app.ActivityThread.performLaunchActivity(ActivityThread.java) 
                                                                          at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java) 
                                                                          at android.app.ActivityThread.-wrap12(ActivityThread.java) 
                                                                          at android.app.ActivityThread$H.handleMessage(ActivityThread.java) 
                                                                          at android.os.Handler.dispatchMessage(Handler.java) 
                                                                          at android.os.Looper.loop(Looper.java) 
                                                                          at android.app.ActivityThread.main(ActivityThread.java) 
                                                                          at java.lang.reflect.Method.invoke(Native Method) 
                                                                          at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java) 
                                                                          at com.android.internal.os.ZygoteInit.main(ZygoteInit.java) 

```

**Would you like to work on the issue?**

I like to. But if anyone wants to work on it, they can.

--- a/app/src/main/java/org/fossasia/phimpme/leafpic/activities/LFMainActivity.java
+++ b/app/src/main/java/org/fossasia/phimpme/leafpic/activities/LFMainActivity.java
@@ -222,6 +222,7 @@ public class LFMainActivity extends SharedMediaActivity {
         albumsMode = true;
         editMode = false;
         securityObj = new SecurityHelper(LFMainActivity.this);
+        if (getIntent().getExtras()!=null)
         pickMode = getIntent().getExtras().getBoolean(SplashScreen.PICK_MODE);
         listAll = StorageProvider.getAllShownImages(LFMainActivity.this);
 

